furusiya/main_interface.py

```diff
@@ -33,13 +37,22 @@ def run(cls, to_run):
             panel=(SCREEN_WIDTH, PANEL_HEIGHT),
             fps_limit=LIMIT_FPS
         )
+        cls.saveload = SaveLoad(cls)
+
+        from model.key_binder import KeyBinder
```

Comment:
Imports should always be at the top of the file.

Reply:
Sadly, having this at the top of the file resulted in cyclic imports.
I'll have another go at rearranging things.

Reply:
That sucks. I never figured out how to get around that. No worries, you can leave it as-is, but please add a comment stating why it's here (cyclical imports)